zndraw/zndraw.py

```diff
@@ -196,9 +197,9 @@ def __delitem__(self, index: int | slice | list[int]):
     def _repr_html_(self):
         from IPython.display import IFrame
 
-        address = f"{self.url}/token/{self.token}"
         # TODO: save address and do not replace in post_init
-        address = address.replace("ws", "http")
+        parsed_url = urllib.parse.urlparse(f"{self.url}/token/{self.token}")
+        address = parsed_url._replace(scheme="http").geturl()
```

Comment:
Is there a way to rename the schema without using the "private" `_replace` method?